plotsearch/serializers.py

```diff
@@ -63,6 +63,7 @@ class Meta:
             "id",
             "name",
             "show_district",
+            "target_selection",
```

Comment:
Seems OK as is. Though, it'd probably be nice to update the fixtures in this commit as well to set this to true in the intended type by default (and as I checked their current state, they weren't updated for `show_district` before either).